rotkehlchen/db/updates.py

```diff
@@ -154,6 +155,53 @@ def update_rpc_nodes(self, to_version: int) -> None:
             new_default_nodes=new_default_nodes,
         )
 
+    def update_contracts(self, to_version: int) -> None:
+        """Updates evm contracts and ABIs in globaldb"""
+        updated_contracts: dict
+        success, updated_contracts = self._get_file_content(  # type: ignore[assignment]  # updated_contracts is a dictionary for sure  # noqa: E501
+            file_url=CONTRACTS_URL.format(branch=self.branch, version=to_version),
+            update_type=UpdateType.CONTRACTS,
+        )
+        if success is False:
+            return
+
+        log.info(f'Applying update for contracts to v{to_version}')
+        new_abi_data = []
+        for single_abi_data in updated_contracts['abis_data']:
+            new_abi_data.append((
+                single_abi_data['id'],
+                single_abi_data['value'],
+                single_abi_data['name'],
+            ))
+
+        new_contracts_data = []
+        for single_contract_data in updated_contracts['contracts_data']:
+            new_contracts_data.append((
+                single_contract_data['address'],
+                single_contract_data['chain_id'],
+                single_contract_data.get('name'),
+                single_contract_data['abi'],
+                single_contract_data.get('deployed_block'),
```

Comment:
Shouldn't the schema be consistent? I would provide all the fields in the json data and set them to null there

Reply:
Okay